### ISSUE TITLE
User Dashboard: Repository overview not showing all repositories

### ISSUE THREAD

**Issue by Wyall** [go-gitea/gitea#12051]
- Gitea version (or commit ref): 1.12.1 Docker
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant

## Description

I have 3 Repos, all are not archived. It seems like the filter settings in the repository view on the top right are misbehaving. See the screenshots. I would expect that all 3 show up in the first picture, but only 2 do. But maybe I am missing something and this works as intended. All 3 show up with the settings in the third screenshot.

## Screenshots

![1](https://user-images.githubusercontent.com/4127751/85697064-c2c3ed00-b6d9-11ea-8c1d-b8a49ce79ea9.png)
![2](https://user-images.githubusercontent.com/4127751/85697074-c3f51a00-b6d9-11ea-81e3-d1864fa522a8.png)
![3](https://user-images.githubusercontent.com/4127751/85697077-c5264700-b6d9-11ea-8707-8a70c90e5d32.png)

**Comment by jangxx** [comment]
I had the the same problem just now. Apparently, for some of my repos, the `is_archived` column in the database was set to `NULL` instead of `0`. I fixed it by simply running 
```sql
UPDATE gitea.repository SET is_archived=0 WHERE is_archived IS NULL
```
on the database and now the repo overview is working again.

**Comment by 6543** [comment]
there is a migration task wo take care of when migrating to v1.13.0 and `gitea doctor` is also able to fix this :) right with gitea >= 1.12.0

https://github.com/go-gitea/gitea/pull/11859

**Comment by 6543** [comment]
@Wyall is is hard to find reverences to the doctor comand?

**Comment by 6543** [comment]
`./gitea doctor --run check-db-consistency --fix` would do it

**Comment by warnat** [comment]
seems like it will be/was fixed with one of the next migrations here  #11853
and here : https://github.com/go-gitea/gitea/commit/b9e281265ee2c510199cd0241ae02bff2bd8ecc1

So, if the problem itself is not acute right now, please close.

**Comment by Wyall** [comment]
Then let's see
